ping-hosts-on-vlan-action-pack/ping-hosts-on-vlan/script.py

```diff
@@ -0,0 +1,68 @@
+import ipaddress
+
+def string_to_list(string_to_convert):
+    numbers = []
+    segments = [ segment.strip() for segment in string_to_convert.split(",") ]
+    for segment in segments:
+        if "-" in segment:
+            for i in range(int(segment.split("-")[0]), int(segment.split("-")[1]) + 1):
+                numbers.append(i)
+        else:
+            numbers.append(int(segment))    
+    return numbers
+
+switch = ctx.getDevice()
+vlan_ids = ctx.changeControl.args['VLAN IDs']
+ctx.alog("{}".format(ctx.alog(vlan_ids)))
+
+if vlan_ids is not None:
+    vlan_ids = string_to_list(vlan_ids)
+else:
+    ctx.alog("No VLAN IDs provided")
+
+for vlan_id in vlan_ids:
+    ctx.alog("Getting interface info on {} for Vlan {}".format(switch.ip, vlan_id))
+    ctx.alog("show ip interface vlan {} output: {}".format(vlan_id, ctx.runDeviceCmds(["show ip interface vlan {}".format(vlan_id)])))
```

Comment:
Line 25 and 27 break the hard limit here, but ideally we want to reduce all lines over 100 characters to 100 or less